From 293452c963188666dae99521294f09a0cf9582e2 Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex.kanavin@gmail.com>
Date: Fri, 4 Aug 2017 16:01:11 +0300
Subject: [PATCH] Do not disable gobject introspection when cross-compiling.

Introspection does work fine for instance in Open Embedded,
one of the most prominent cross-compilation frameworks
(through qemu emulating target hardware), so let the user
decide if he wants the feature or not.

Upstream-Status: Pending
Signed-off-by: Alexander Kanavin <alex.kanavin@gmail.com>
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 43cbfd9..8a32f26 100644
--- a/meson.build
+++ b/meson.build
@@ -147,7 +147,7 @@ root_dir = include_directories('.')
 
 gnome = import('gnome')
 gir = find_program('g-ir-scanner', required: false)
-build_gir = gir.found() and not meson.is_cross_build() and not get_option('disable_introspection')
+build_gir = gir.found() and not get_option('disable_introspection')
 
 subdir('json-glib')
 
-- 
2.13.2

